packages/ffe-datepicker-react/src/datepicker/Datepicker.spec.js

```diff
@@ -410,6 +410,25 @@ describe('<Datepicker />', () => {
         });
     });
 
+    describe('should have element references in rendered components', () => {
+        const TestUtils = require('react-dom/test-utils');
```

Comment:
Can you please hoist import to the top of the test file?

Also, why would you need these test utils? We're already using Enzyme, and you should have all the query tools need available there.